docs/source/index.rst

```diff
@@ -1,40 +1,48 @@
-.. mlchecks documentation master file, created by
-   sphinx-quickstart on Sun Oct 24 10:19:19 2021.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+=====================================
+Welcome to Deepchecks!
+=====================================
+Deepchecks is the leading tool for validation you machine learning models
+and data and enables doing so with minimal effort. Deepchecks accompanies you through
+various validation needs such as verifying your data's integrity, inspecting its distributions,
+validating data aplits, evaluating your model and comparing between different models.
 
+Head over to the :doc:`getting-started/index` guide to get it up and running.
 
-Welcome to Deepchecks' documentation!
-=====================================
 
+.. note::
+    Welcome! In addition to perusing the documentation, please feel free to
+    to ask questions on our `Slack channel <https://join.slack.com/t/deepcheckscommunity/shared_invite/zt-y28sjt1v-PBT50S3uoyWui_Deg5L_jg>`__, or to post a issue or start a discussion on `Github <https://github.com/deepchecks/deepchecks/>`__.
+
+
+Documentation Structure
+=========================
 
 .. toctree::
-   :maxdepth: 2
-   :hidden:
+    :maxdepth: 2
 
-   getting-started/index
+    getting-started/index
 
 
 .. toctree::
-   :maxdepth: 2
+    :maxdepth: 2
 
-   examples/index
+    examples/index
 
 
 .. toctree::
-   :maxdepth: 2
+    :maxdepth: 2
 
-   user-guide/index
+    user-guide/index
 
 
 .. toctree::
-   :maxdepth: 3
-
-   api/index
+    :maxdepth: 3
 
+    api/index
 
-Indices and tables
-~~~~~~~~~~~~~~~~~~~~
 
+Reference Indices
+====================
+* :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
```

Comment:
Do we really need this in our docs?

Reply:
Thought it may be useful (an specifically more useful than the two other things there that are already very accessible from the documentation), don't you think?
It's just an option to see (and filter by letters) "everything" - thought it's mainly useful if the search doesn't put in the top options the specific thing you're looking for

Reply:
removed all of them